screen/reports/src/main/java/com/ivy/reports/ReportViewModel.kt

```diff
@@ -389,7 +389,24 @@ class ReportViewModel @Inject constructor(
             transactionRepository.findAll()
         }
 
+        val exacludeableByTagTransactionsIds = if (filter.excludedTags.isNotEmpty()) {
+            tagRepository.findByAllAssociatedIdForTagId(filter.excludedTags)
+                .asSequence()
+                .flatMap { it.value }
+                .map { TransactionId(it.associatedId.value) }
+                .distinct()
```

Comment:
not sure if it will have any benefit but it will be better to do `.distinct()` before creating `TransactionId` objects. Somewhat like:
```
.asSequence()
.flatMap { it.value.associatedId.value }
.distinct()
.map { TransactionId(it) }
.toList()
```

Reply:
Done